js/ad_manager_vast.js

```diff
@@ -1788,6 +2097,9 @@ OO.Ads.manager(function(_, $) {
         result.maintainAspectRatio = nonLinear.attr("maintainAspectRatio");
         result.minSuggestedDuration = nonLinear.attr("minSuggestedDuration");
         result.nonLinearClickThrough = nonLinear.find("NonLinearClickThrough").text();
+        result.nonLinearClickTracking = filterEmpty($(nonLinearAdsXml).
```

Comment:
Double checked, $.find() returns [] if it does not find the selector/element. Then, calling map() on an empty array should do nothing ~ gracefully